rackcity/api/serializers/asset_serializers.py

```diff
@@ -546,7 +546,12 @@ def get_blades_in_chassis_cp(asset_cp):
 def generate_network_graph(asset):
     try:
         nodes = []
-        nodes.append({"id": asset.id, "label": asset.hostname})
+        if hasattr(asset, "related_asset") and asset.related_asset:
+
+            route_id = asset.related_asset.id
+        else:
+            route_id = asset.id
+        nodes.append({"id": asset.id, "route_id": route_id, "label": asset.hostname})
```

Comment:
i do this everywhere -> that way route_id is always the id of the live asset if it exists